api/pkg/service/resource/resource.go

```diff
@@ -208,18 +211,25 @@ func initResource(r model.Resource) *resource.Resource {
 	return res
 }
 
-func minVersionInfo(r model.ResourceVersion) *resource.Version {
+func tinyVersionInfo(r model.ResourceVersion) *resource.MinVersionInfo {
```

Comment:
Any reason why the name is changed to tinyVersionInfo when it returns MinVersionInfo?

Reply:
As it returns only the `version-id` and the `version` of the resource hence added a function called `tinyVersionInfo`  and the `minVersionInfo` calls the `tinyVersionInfo` to get the version-id and version